src/style.css

```diff
@@ -0,0 +1,87 @@
+/* #region page */
+html {
+  --search-icon-big: url(images/Search.svg) 26px 50% no-repeat;
+  --search-icon-small: url(images/Search.svg) 13px 50% no-repeat;
+  --linear-gradient-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  --font-color: #3d4e61;
+  --font-size-big: 16px;
+  --font-size-small: 14px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+/* #endregion */
+/* #region main */
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments)

Reply:
Noted! As for the comments, I just liked how Misha used sections in layout_landing-page task :)